doc/design/volumes/nfsvolumestores.md

```diff
@@ -18,10 +21,36 @@ Allow the container user to
  2. Exposing shared storage configuration via VIC (e.g. IOPS per client, storage policy, etc.)
  3. Management of shared storage via VIC (e.g. container quiesce for storage maintenance, quota manipulation of the target, etc.)
 
+### Networking
+
+Containers will need to be on an appropriate network for the NFS volume store to be accessible.
+
+There are ways this could be done:
+ - allow association of volume-store with container-network to allow direct coupling
```

Comment:
I would prefer if the vic user is responsible for tracking this. All that should be required is that the container using the nfs store is connected to the correct network(s). We can validate this during container/tether bring up and communicate it to the caller.